tests/cpgrid/zoltan_test.cpp

```diff
@@ -89,7 +89,7 @@ BOOST_AUTO_TEST_CASE(zoltan)
     int m_argc = boost::unit_test::framework::master_test_suite().argc;
     char** m_argv = boost::unit_test::framework::master_test_suite().argv;
     {
-        auto& inst = Dune::MPIHelper::instance(m_argc, m_argv);
+        const auto& inst = Dune::MPIHelper::instance(m_argc, m_argv);
```

Comment:
If you're changing this anyway, maybe we could just tag this as `[[maybe_unused]]` to remove the cast below?